app/views/layouts/_footer.html.erb

```diff
@@ -0,0 +1,12 @@
+<footer class="footer">
+  <small>
+    <%= t "Author_html" %>
```

Comment:
tên biến ko bắt đầu với chữ cái viết hoa 

self check toàn bộ file html bên dưới